target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -658,6 +658,9 @@ ramips_board_detect() {
 	*"Y1S")
 		name="y1s"
 		;;
+	*"YK2")
```

Comment:
This board name detection method is deprecated.
OpenWrt introduced another detection based on device-tree compatible string.By using that, you don't need to edit this file and the board name got automatically set to the first compatible string in dts. (In your case it's youku,yk2)
You could check some recently added device support in ramips for details.

Reply:
ok, I removed it.